server/src/resolvers.js

```diff
@@ -34,6 +34,19 @@ export const Resolvers = {
       console.log(args);
       return userHandler.createUserX(_, args, ctx);
     },
+    deleteUser(_, DeleteUserInput, ctx) {
```

Comment:
I know you're trying to flesh the API out but I don't see a use for this. I see 2 situations where we would delete a user:
1. admin user needs to delete a user - input would be an ID number and honestly is probably just going to do it via SQL.
2. User wants to delete themselves. No ID number needed because their auth token identifies them.

Reply:
Testing.

I cant test that the code to create and catch duplicate users works if I cant delete the user programatically. I was having to flush the DB by hand every time i ran a of the signup mutation.